clients/src/main/java/org/apache/kafka/clients/ClientDnsLookup.java

```diff
@@ -22,7 +22,8 @@
 
     DEFAULT("default"),
     USE_ALL_DNS_IPS("use_all_dns_ips"),
-    RESOLVE_CANONICAL_BOOTSTRAP_SERVERS_ONLY("resolve_canonical_bootstrap_servers_only");
+    RESOLVE_CANONICAL_BOOTSTRAP_SERVERS_ONLY("resolve_canonical_bootstrap_servers_only"),
+    USE_FIRST_DNS_IPS("use_first_dns_ips");
```

Comment:
In the KIP this field is called `use_first_dns_ip`, I think it was agreed in the discussion to remove the `s`

Reply:
I edited the PR to include this change.

Reply:
@mimaison Based on @ijuma 's suggestion, I am not adding `use_first_dns_ip`. I am going to deprecate `default` as the value and make `use_all_dns_ips` as the default value.

Reply:
I agree it makes sense to not introduce this new value. It's unfortunate the old value is called `default` but hopefully we'll remove it in the next major release.

Reply:
ok. resolving this conversation.